ci/travis_pypi_setup.py

```diff
@@ -9,7 +9,6 @@
 import json
 import os
 from getpass import getpass
-import yaml
 from cryptography.hazmat.primitives.serialization import load_pem_public_key
```

Comment:
why line is load_pem_public_key and default_backend on?  i cant find them outside of the removed lines?